src/array.ts

```diff
@@ -68,6 +69,11 @@ export function from<T>(arrayLike: ArrayLike<T>, mapFunction?: MapCallback<T>, t
  * @return The new Array
  */
 export function from<T>(arrayLike: (string | ArrayLike<T>), mapFunction?: MapCallback<T>, thisArg?: {}): ArrayLike<T> {
+	// Use the native Array.from() if it exists
+	if (has('array-from')) {
```

Comment:
While we don't have it standardized yet (see dojo/meta#5) I would still like to try to use them so could we do `es6-array-from` *et al.* instead?